internal/builder/release_manifest_reader_test.go

```diff
@@ -142,159 +68,6 @@ version: 1.2.3
 					Expect(err).To(MatchError(ContainSubstring("no such file")))
 				})
 			})
-
```

Comment:
Regardless of who's doing the work if we require certain behavior to exist it should be reflected in the caller's test coverage too. But pragmatically speaking I'm onboard here.